### ISSUE TITLE
Can't load version 2.1.1: Module has no exports

### ISSUE THREAD

**Issue by oanguenot** [Makanz/chartjs-plugin-trendline#90]
Hi,
I updated from version 2.1.0 to version 2.1.1 and I have now the following issue:

```shell
export 'default' (imported as 'chartTrendline') was not found in 'chartjs-plugin-trendline' (module has no exports)
```

Installed from yarn/npm

I use the following:

```js
import Chart from "chart.js/auto";
import {DateTime} from "luxon";
import zoomPlugin from "chartjs-plugin-zoom";
import chartTrendline from 'chartjs-plugin-trendline';
import annotationPlugin from 'chartjs-plugin-annotation';
```

Versions used:
-  "chart.js": "4.4.3",
-  "chartjs-adapter-luxon": "1.3.1",
-  "chartjs-plugin-annotation": "^3.0.1",
-  "chartjs-plugin-streaming": "2.0.0",
-  "chartjs-plugin-trendline": "2.1.1",
-  "chartjs-plugin-zoom": "2.0.1",

Thanks

**Comment by Makanz** [comment]
try
` import pluginTrendlineLinear from 'chartjs-plugin-trendline';`

**Comment by oanguenot** [comment]
Sorry, same problem occured

**Comment by sammcj** [comment]
Yeah seems broken for me too?

> try `import pluginTrendlineLinear from 'chartjs-plugin-trendline';`

Error chartjs-plugin-trendline has no default export

**Comment by Makanz** [comment]
I have released a new version 2.1.2. I hope it will fix it.

**Comment by oanguenot** [comment]
Yes, thank you, problem is fixed on my side
